pkg/virt-api/rest/authorizer.go

```diff
@@ -50,6 +50,8 @@ type VirtApiAuthorizor interface {
 	GetGroupHeaders() []string
 	AddExtraPrefixHeaders(header []string)
 	GetExtraPrefixHeaders() []string
+	NewSubjectAccessReview(*restful.Request) (*authv1.SubjectAccessReview, error)
```

Comment:
Can you squash this commit with the one where these changes are useful? Then it will be easier to review.
This commit by itself does not seem useful.

Reply:
Done